### ISSUE TITLE
Trailing space on URLs returns errors in results

### ISSUE THREAD

**Issue by graphiclunarkid** [openrightsgroup/cmp-issues#68]
If I leave a trailing space after the URL (eg "indiewebcamp.com ") I see a list of errors, if I delete it I don't. Autocomplete on mobile will often append a space, can we strip() the URL field?

**Comment by graphiclunarkid** [comment]
Fixed by trim() ing the URL in the SubmitURL snippet. Requested testing from the person who reported the bug.

We should consider doing this in the API so that we don't have to worry about each client individually though.

**Comment by dantheta** [comment]
Yes, very strange that PHP's parse_url function allowed this.  The API is a good place to fix it.

**Comment by graphiclunarkid** [comment]
I commented out my fix in the SubmitURL snippet to test the change in the API code. I observed different results for http://www.example.com/ with and without a trailing space character. With a trailing space, fewer results are returned than without.

If I trim() the URL in the SubmitURL snippet the results for both are identical.

Could the change have been made in the wrong place? Or alternatively, has this version of the code not yet been deployed to the server?

I have left the SubmitURL solution in place for now.
